admin/admin-service/src/main/java/io/mosip/admin/util/Utility.java

```diff
@@ -0,0 +1,88 @@
+package io.mosip.admin.util;
+
+import io.mosip.kernel.core.util.StringUtils;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+
+@Component
+public class Utility {
+
+    private static final Logger logger = LoggerFactory.getLogger(Utility.class);
+
+    @Value("${mosip.kernel.config.server.file.storage.uri}")
+    private String configServerFileStorageURL;
+
+    @Value("${mosip.admin.identityMappingJson}")
+    private String residentIdentityJson;
```

Comment:
Since it is not resident module, can we rename this variable to just identityMappingJson ?

Reply:
taken care

Reply:
Resolved